hathor/transaction/block.py

```diff
@@ -356,10 +295,8 @@ def get_base_hash(self) -> bytes:
         return sha256d_hash(self.get_header_without_nonce())
 
     def get_height(self) -> int:
```

Comment:
I think my initial idea was to delete this method and change everywhere to use `static_metadata.height`, but then I noticed too many places used it and decided that keeping the convenience method was better for now. I still think it could be deleted in the future, though. See the [comment above](https://github.com/HathorNetwork/hathor-core/pull/1014#discussion_r1715802174) for a related discussion.

Reply:
Got it. Thanks!